### PR TITLE
added

### PR DESCRIPTION
**Pull request by cuttty**

### CODE REVIEW

**Comment by cuttty** [comment]
no comments

**Comment by cuttty** [comment]
лишняя строчка - 28, я уже потом заметила

File: task4/FrolovaS.c [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            printf("%d ", a[i][j]);
+        }
+        printf("\n");
+        
+    }
+    if (res==0) {
+        printf ("no ways");
+    }
+    else {
+        printf ("there is a way");
```
Comment (lordoz234) [inline task4/FrolovaS.c:58]: We should print path, if path exist

Comment (cuttty) [reply task4/FrolovaS.c:58]: you can see a path, the program printed it, but it also printed a message about availability of the path

**Comment by lordoz234** [comment]
LGTM!

**Pull request merged**